kubernetes/docker/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# This script can be used to build the custom docker images required
+# for Kubernetes. This involves running the entire OpenWhisk gradle
+# build process and then creating the custom images for OpenWhisk.
+
+# prerequisites:
+#   * be able to run `cd <home_openwhisk> ./gradlew distDocker`
+
+set -ex
+
+
+if [ -z "$1" ]; then
+cat <<- EndOfMessage
+  First argument should be location of which docker repo to push all
+  of the built OpenWhisk docker images. This way, Kubernetes can pull
+  any images it needs to.
+EndOfMessage
+
+exit 1
+fi
+
+OPENWHISK_DIR=""
+if [ -z "$2" ]; then
+cat <<- EndOfMessage
+  Second argument should be the location of where the OpenWhisk repo lives.
+  By default the location is $HOME/workspace/openwhisk
+EndOfMessage
+
+  OPENWHISK_DIR=$HOME/workspace/openwhisk
+else
+  OPENWHISK_DIR="$2"
+fi
+
+pushd $OPENWHISK_DIR
+  ./gradlew distDocker
+popd
+
+## Retag new images for public repo
+docker tag whisk/badaction "$1"/whisk_badaction
```

Comment:
Minor thing but you may find it easier to write this as something like:
```
for name in badproxy cli example ....
do
  docker tag whisk/$name "$1"/whisk_${name}
  docker push "$1"/whisk_${name}
done
```
up to you.